app/services/bgs/dependent_service.rb

```diff
@@ -2,13 +2,60 @@
 
 module BGS
   class DependentService
-    def get_dependents(current_user)
-      service = BGS::Services.new(
-        external_uid: current_user.icn,
-        external_key: current_user.common_name
+    include SentryLogging
+
+    def initialize(user)
+      @user = user
+    end
+
+    def get_dependents
+      service.claimants.find_dependents_by_participant_id(@user.participant_id, @user.ssn)
+    end
+
+    def submit_686c_form(claim)
+      va_file_number_with_payload = add_va_file_number_to_payload
+
+      VBMS::SubmitDependentsPDFJob.perform_async(claim.id, va_file_number_with_payload)
```

Comment:
You're calling the variable as, "va_file_number_with_payload" but when you read it from the queue (and in your rspec) you call it "vet_info". I don't think the abstraction is quite right.

I'm not certain what the right approach is, but took a quick crack at one...
```ruby
bgs_person = service.people.find_person_by_ptcpnt_id(@user.participant_id)
vet_info = VetInfo.new(@user, bgs_person)
VBMS::SubmitDependentsPDFJob.perform_async(claim.id, vet_info.to_686c_form_hash)

### vet_info.rb
class VetInfo
  def initialize(user, bgs_person)
    @user = user
    @bgs_person = bgs_person
  end

  def to_686c_form_hash
    {
      'veteran_information' => {
        'full_name' => {
          'first' => @user.first_name,
          'middle' => @user.middle_name,
          'last' => @user.last_name
        },
        'ssn' => @user.ssn,
        'va_file_number' => @bgs_person[:file_nbr],
        'birth_date' => @user.birth_date
      }
    }
  end
end
```